WordPressVIPMinimum/Sniffs/Compatibility/RobotstxtSniff.php

```diff
@@ -31,22 +31,19 @@ public function register() {
 	/**
 	 * Processes the tokens that this sniff is interested in.
 	 *
-	 * @param File $phpcsFile The PHP_CodeSniffer file where the token was found.
-	 * @param int  $stackPtr  The position in the stack where the token was found.
+	 * @param int $stackPtr  The position in the stack where the token was found.
```

Comment:
There are still two spaces before the description (for each of these instances), which `WordPress-Docs` may complain about.